tests/unit_tests/flow_variable_tests/CMakeLists.txt

```diff
@@ -78,3 +78,82 @@ foreach(dim RANGE 2 3)
 
 endforeach()
 
+set(TEST_SRC
+auxiliary_equations_int_by_parts.cpp
+    )
+
+foreach(dim RANGE 1 1)
```

Comment:
Would it work for 2 and 3 dimensions?

Reply:
Yes the next one in that file is for 2D and 3D. It was to use mpimax in ctest for 2D and 3D, and mpirun -n 1 for the 1D case.

Reply:
I see. Take a look at how it is done here:
https://github.com/dougshidong/PHiLiP/blob/10ff982e8144b939d1eedd8c465ee9e66e900c31/tests/unit_tests/sensitivities/CMakeLists.txt#L80-L87

Good rule of thumb: if you copy paste, there is a better way.

Reply:
I actually had already addressed this in a commit I made before the force push